### PR TITLE
[macOS] Remove macOS 10.13 related code in the VE repo

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
Since macOS 10.13 pool have been deprecated, we need to remove all the related code in the VE repo

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/provision/bootstrap-provisioner/change_password [hunk 1]
```diff
@@ -8,11 +8,7 @@ export PATH=/usr/bin:/usr/sbin:/usr/local/bin:/bin:/sbin
 
 macosver="$(sw_vers | grep ProductVersion | awk {'print $2'})"
 
-if [[ $macosver =~ 10.13.* ]]; then
-    sudo /usr/bin/dscl . -passwd /Users/$USERNAME "$NEW_PASSWD"
-else
-    sudo /usr/sbin/sysadminctl -resetPasswordFor $USERNAME -newPassword "$NEW_PASSWD" -adminUser $USERNAME -adminPassword "$OLD_PASSWD"
-fi
+sudo /usr/sbin/sysadminctl -resetPasswordFor $USERNAME -newPassword "$NEW_PASSWD" -adminUser $USERNAME -adminPassword "$OLD_PASSWD"
```
Comment (al-cheb) [inline images/macos/provision/bootstrap-provisioner/change_password:11]: Let's revert changes in this file,

File: images/macos/provision/core/firefox.sh [hunk 2]
```diff
@@ -4,17 +4,11 @@ source ~/utils/utils.sh
...
-    brew_smart_install "geckodriver"
-    geckoPath="$(brew --prefix geckodriver)/bin"
-else
-    geckoUrl="$(curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest | jq -r '.assets[].browser_download_url | select(endswith("macos.tar.gz"))')"
-    download_with_retries $geckoUrl "/tmp" "geckodriver.tar.gz"
-    geckoPath="/usr/local/bin"
-    tar -xzf /tmp/geckodriver.tar.gz -C $geckoPath
-fi
+echo "Installing Geckodriver..."
+geckoUrl="$(curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest | jq -r '.assets[].browser_download_url | select(endswith("macos.tar.gz"))')"
```
Comment (al-cheb) [inline images/macos/provision/core/firefox.sh:8]: We should install gecko from brew: `brew_smart_install "geckodriver"`

**Review by vsafonkin — DISMISSED** [review]

**Review by miketimofeev — APPROVED** [review]

File: images/macos/provision/core/chrome.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ brew install --cask chromedriver
 echo "Installing Selenium"
 brew_smart_install "selenium-server"
 
-CHROMEWEBDRIVER_DIR=$(readlink $(which chromedriver) | xargs dirname)
-echo "export CHROMEWEBDRIVER=$CHROMEWEBDRIVER_DIR" >> "${HOME}/.bashrc"
+echo "Installing Geckodriver"
```
Comment (al-cheb) [inline images/macos/provision/core/chrome.sh:13]: Why do we put geckodriver in the chrome.sh script?

File: images/macos/provision/core/firefox.sh (hunk 2 above)
Comment (al-cheb) [reply images/macos/provision/core/firefox.sh:8]: Why have you marked as resolved?

Comment (dibir-magomedsaygitov) [reply images/macos/provision/core/firefox.sh]: @al-cheb sorry, fixed now.

File: images/macos/provision/core/chrome.sh (hunk 3 above)
Comment (dibir-magomedsaygitov) [reply images/macos/provision/core/chrome.sh]: Thank you, fixed.

**Review by miketimofeev — APPROVED** [review]

File: images/macos/provision/core/chrome.sh [hunk 4]
```diff
@@ -13,4 +13,4 @@ brew_smart_install "selenium-server"
 CHROMEWEBDRIVER_DIR=$(readlink $(which chromedriver) | xargs dirname)
```
Comment (al-cheb) [inline images/macos/provision/core/chrome.sh:13]: Let's revert any changes in this script.

File: images/macos/provision/core/firefox.sh [hunk 5]
```diff
@@ -4,17 +4,8 @@ source ~/utils/utils.sh
...
-    brew_smart_install "geckodriver"
-    geckoPath="$(brew --prefix geckodriver)/bin"
-else
-    geckoUrl="$(curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest | jq -r '.assets[].browser_download_url | select(endswith("macos.tar.gz"))')"
-    download_with_retries $geckoUrl "/tmp" "geckodriver.tar.gz"
-    geckoPath="/usr/local/bin"
-    tar -xzf /tmp/geckodriver.tar.gz -C $geckoPath
-fi
+echo "Installing Geckodriver..."
+brew_smart_install "geckodriver"
```
Comment (al-cheb) [inline images/macos/provision/core/firefox.sh:8]: Where is geckoPath?

**Review by MaksimZhukov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**